pyzeebe/worker/task_state.py

```diff
@@ -7,19 +7,18 @@
 
 class TaskState:
     def __init__(self):
-        self._active_jobs = set()
+        self._active_jobs = list()
```

Comment:
Because a list is now used, `remove` will only remove one of the occurrences (in case there's now a duplicate in the list). It also makes it less reliable (for future use), as there may now be duplicate instances. 

In my opinion, this could be kept as a set, and in the `add` method the `ValueError` could simply be passed/logged.

Reply:
But this is the expected behavior. If a job gets reactivated, the worker should work on them both.